### DISCUSSION TITLE
Difference between System Informer and Windows Task Manager CPU usage

### DISCUSSION THREAD

**Discussion by trparky** [winsiderss/systeminformer#1450] [Questions / Answers]
System Informer is reporting a lower CPU usage percentage than that of the default Windows 11 Task Manager. Is there a reason why?

**Comment by Masamune3210** [comment]
Difference in methods used to calculate CPU use, they are both correct just using different metrics

**Reply by Masamune3210** [reply]
IIRC Task Manager does a lot of things like averaging all the use between cores

**Reply by ge0rdi** [reply]
Task Manager shows CPU _utility_ instead of _utilization_:
https://learn.microsoft.com/en-US/troubleshoot/windows-client/performance/cpu-usage-exceeds-100

More on it also here:
https://aaron-margosis.medium.com/task-managers-cpu-numbers-are-all-but-meaningless-2d165b421e43
